extensions/wikia/CuratedContent/CuratedContentValidator.class.php

```diff
@@ -63,7 +70,20 @@ private function validateImage( $sectionOrItem ) {
 		}
 	}
 
-	private function validateSection( $section ) {
+	public function validateSectionItems( $section ) {
+		if ( !empty( $section['items'] ) && is_array( $section['items'] ) ) {
+			foreach ($section['items'] as $item) {
+				$this->validateSectionItem( $item );
```

Comment:
Here we use `validateSectionItem` and in the `validateFeaturedSectionItems` we use `validateItem`. This is confusing :see_no_evil: